app/models/trustee.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class ElectionTrustee < ApplicationRecord
+  has_many :election
```

Comment:
As stated before, by convention it should be in plural:

```suggestion
  has_many :elections
```